src/internet_identity/src/main.rs

```diff
@@ -411,6 +411,11 @@ fn encode_metrics(w: &mut MetricsEncoder<Vec<u8>>) -> std::io::Result<()> {
             s.storage.borrow().user_count() as f64,
             "Number of users registered in this canister.",
         )?;
+        w.encode_gauge(
+            "internet_identity_max_users",
+            s.storage.borrow().max_users() as f64,
```

Comment:
Wouldn’t it be better to return the `hi-lo`?

Reply:
Yes, that gives us more information.

Reply:
I guess my point was that we want to get the correct number, even if the canister is configured differently during init. But yeah, we can do the subtraction in grafana if the concrete bounds are useful (which they may be later)